Module4_Labs/Lab2_Doubly_Linked_List/Cards/2.md

```diff
@@ -8,4 +8,6 @@
 
 You can traverse the list to get each element of the node and do something with it, either printing each element in the list or comparing it with a specific value.
 
-Under your doubly linked list class, define a function call **printList()**. It should traverse the list and print each element separated by a comma.
\ No newline at end of file
+Under your doubly linked list class, define a function call **printList()**. It should traverse the list and print each element separated by a comma.
+
+Note that for edge cases, like when the list is empty, your code may throw an error. Try using a checkpoint for certain cases before going into the list traversal code.
```

Comment:
"Try using a checkpoint for certain cases before going into the list traversal code."
use more technical word than "checkpoint"